app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -61,20 +64,36 @@ export const ViewContent: React.FC = () => {
 
   const path = content?.fileReferences ? content?.fileReferences[0]?.path : '';
 
-  const myMinister = localStorage.getItem('myMinister');
+  const myMinisters = userInfo?.preferences?.myMinisters;
 
   React.useEffect(() => {
-    // this will bold the ministers name in the summary or body, only when viewing from the my minister list
-    const regex = new RegExp(myMinister ?? '', 'gi');
-    if (window.location.href.includes('my-minister')) {
-      if (content?.summary && !content.summary.includes(`<b>${myMinister}</b>`))
-        setContent({ ...content, summary: content.summary.replace(regex, `<b>${myMinister}</b>`) });
+    // this will bold the ministers name in the summary or body, only when viewing from the my minister list when the content has been received
+    if (window.location.href.includes('my-minister') && !boldingComplete && !!content) {
+      let tempBody = content?.body;
+      let tempSummary = content?.summary;
+      myMinisters.length &&
+        myMinisters.forEach((minister: string, index: number) => {
+          const regex = new RegExp(minister ?? '', 'gi');
+          console.log(minister);
+          if (content?.summary && !content.summary.includes(`<b>${minister}</b>`)) {
+            tempSummary = tempSummary?.replace(regex, `<b>${minister}</b>`);
+          }
 
-      if (content?.body && !content.body.includes(`<b>${myMinister}</b>`)) {
-        setContent({ ...content, body: content.body.replace(regex, `<b>${myMinister}</b>`) });
-      }
+          if (
+            content?.body &&
+            !content.body.includes(`<b>${minister}</b>`) &&
+            content.body.includes(minister)
+          ) {
+            tempBody = tempBody?.replace(regex, `<b>${minister}</b>`);
+          }
+
+          if (index === myMinisters.length - 1) {
```

Comment:
Have to use the temp variables as scope requires it (otherwise will overwrite with last replacement in the list), also only once we are done iterating the user's ministers is when we want to update the content